01-Login/README.md

```diff
@@ -1,50 +1,73 @@
-# Auth0 Vue.js Login
+# Scenario #1 - Logging In and Gated Content
 
-This sample demonstrates how to add authentication to a Vue.js application with Auth0. The sample makes use of Auth0's hosted login page which provides centralized authentication.
+This sample demonstrates:
 
-## Getting Started
+- Logging in to Auth0 using Redirect Mode
+- Accessing profile information that has been provided in the ID token
+- Gated content. The `/profile` route is not accessible without having first logged in
 
-If you haven't already done so, [sign up](https://auth0.com) for your free Auth0 account and create a new client in the [dashboard](https://manage.auth0.com). Find the **domain** and **client ID** from the settings area and add the URL for your application to the **Allowed Callback URLs** box.
+## Configuration
 
-Clone the repo or download it from the Vue.js quickstart page in Auth0's documentation.
+The project needs to be configured with your Auth0 domain and client ID in order for the authentication flow to work.
+
+To do this, first copy `src/auth/auth0-variables.sample.js` into a new file in the same folder called `auth0-variables.js`, and replace the values within with your own Auth0 application credentials.
+
+```js
+// src/auth/auth0-variables.js
+
+export default {
+  domain: "<YOUR AUTH0 DOMAIN>",
+  clientId: "<YOUR AUTH0 CLIENTID>",
```

Comment:
```suggestion
  clientId: "<YOUR AUTH0 CLIENT ID>",
```

Reply:
My mistake - these are all wrong anyway, as the Readme needs to be re-written to cater for the new JSON file instead of using the `.env` file.